views/partials/banner_breach.hbs

```diff
@@ -12,5 +12,6 @@
   <span class="breach-info"><span class="medium">Compromised data:</span> {{breachDataClasses breach.meta.DataClasses }}</span>
   <span class="breach-info"><span class="medium">Compromised accounts:</span> {{localeString breach.meta.PwnCount }}</span>
   <span class="breach-info">{{{ breach.meta.Description }}}</span>
+  <span class="source-info">Breach data provided by<span class="demi"> Have I been Pwned?</span></span>
```

Comment:
I think I mentioned this elsewhere, but should we add a link to the HIPB website here?

Reply:
I asked about this and was told that the agreement was to not link. I do think a link here would be helpful and make sense however.